### ISSUE TITLE
Web - Step 5 - Topic - Audit Tools - No Langauge Set for "Verité" in Main Content Section of Page

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#65]
The word "Verité" under the "**Source: Responsible Sourcing Tool 1A...**" heading should be marked up as French. There might be other words that are not English on this page or others that need to be marked up to use the correct language.

Step 5 - Topic - Audit Tools - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/steps/5/topic/3

**Comment by KyleLivingstonDOL** [comment]
We are addressing in another issue.

**Comment by Abychalil** [comment]
lang Attribute set to french at the mentioned places.

fixed

**Comment by jfc3-dol** [comment]
@Abychalil,

Thanks for correcting the language issues for French on the screen. Did you do the same for any other screen that might have French or Spanish on either the English, Spanish, or French pages because this is one of the ones I found and not knowing content can't tell the others?

Will look at it when the next version/build is sent, and I  have time to test.

**Comment by jfc3-dol** [comment]
Thanks for adding the language around the text "Verité".

Why did you make the firs occurrence its own link instead of part of the text of the larger one? Also, you need to include the apostrophe "S" in the French language SPAN for the last on the page.

**Comment by KyleLivingstonDOL** [comment]
Hi John,

Thanks for pointing this out. Can you please elaborate on your comment below? I don't see the reference regarding "the first occurrence it's own link"?

Also, please note in Spanish and French there is no " 's ". That is done by adding "de" in front of the word to make it possessive. Appreciate the head's up, but if we can keep the comments to 508 compliance, that would be really amazing. We want to close out these items. We are working to get all of these items done in a timely manner.

Best,
Kyle

> Thanks for adding the language around the text "Verité".
> 
> Why did you make the firs occurrence its own link instead of part of the text of the larger one? Also, you need to include the apostrophe "S" in the French language SPAN for the last on the page.

**Comment by jfc3-dol** [comment]
@KyleLivingstonDOL,

See the two attached screenshots that show the SPAN tag only put the word Verite in the SPAN and not the apostrophe "S", which should be part of it.

**Highlighted on page**

![Step 5 topic 3 - verite needs to include the apostrophe S](https://user-images.githubusercontent.com/10423487/68343156-2e5d3980-00ba-11ea-9831-ef034fcf6d0f.png)

**Highlighted on both page and code**

![Step 5 topic 3 - verite needs to include the apostrophe S showing code](https://user-images.githubusercontent.com/10423487/68343157-2ef5d000-00ba-11ea-936a-6be48783a65a.png)

**Comment by KyleLivingstonDOL** [comment]
Thanks John. ZTI, can you please review and check whether or not the Span Lang is including the "de vertite" in it? I see that this is under the english version with the Span Lang brackets. Happy to discuss on a call.

**Comment by jfc3-dol** [comment]
All three places"Verité" are in this page now have lang="fr" around them. The page is now the following - https://dolcontentdev.opadev.dol.gov/ilab/complychain/steps/7/topic/3

So I'm going to close this issue because it's coded as expected.
